season/string_manipulation.md

```diff
@@ -19,15 +19,26 @@ iverson_rant = "If I can't practice, I can't practice, man. If I'm hurt, I'm hur
 Programmatically manipulate `iverson_rant` to reflect the output above.
 
 ```ruby
-# write your code here
+  wordsarray = iverson_rant.split
+  first_letters_array = []
+
+  wordsarray.map do |word|
+    first_letters_array.push(word.chr)
+  end 
+
```

Comment:
Looks good! Two little nitpicks:
* Generally in Ruby, you'll want to separate variable names with underscores (e.g. `words_array` instead of `wordsarray`)
* `.map` is going to make a new array entirely, but on line 25, you don't care about making a new array (since you're pushing to a separate array on line 26). I would use `.each` instead of `.map` in this instance; `.each` will execute some code for each item passed to the block (in this case, `word`), but will return the original array.